CHANGELOG.next.asciidoc

```diff
@@ -78,8 +78,15 @@ https://github.com/elastic/beats/compare/v8.2.0\...main[Check the HEAD diff]
 
 *Metricbeat*
 
+
+- update kubernetes apiserver metricset to not collect deprecated metrics and fix dashboard {pull}31973[31973]
```

Comment:
Can you please check why these un-related change logs showing up in addition here?